plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/glue/TestHiveGlueMetastore.java

```diff
@@ -676,6 +694,105 @@ public void testGetPartitionsFilterIsNullWithValue()
                 ImmutableList.of(ImmutableList.of(GlueExpressionUtil.NULL_STRING)));
     }
 
+    @Test
+    public void testUpdateStatisticsOnCreate()
```

Comment:
The problem was with Glue, but perhaps the test could be generic for all metastore impls.
(and disabled in those which don't support stats)

Reply:
Possibly. I would prefer to look into that as a followup.